src/services/routes/org.routes.ts

```diff
@@ -131,5 +131,26 @@ export default function registerRoutes(
     },
   });
 
+  fastify.route({
+    method: ['DELETE'],
+    url: '/orgs/:orgName',
+
+    schema: {
+      params: ORG_ROUTE_PARAMS,
+    },
+
+    async handler(request, reply): Promise<void> {
```

Comment:
I think we should return a 404 when the org isn't found. It's what a RESTFul API is supposed to do and what we've done in other endpoints. If `PATCH /orgs/example.com` results in a 404, why does `DELETE /orgs/example.com` not?